src/store/flow/block-types/MobilePrimitives_SelectManyResponseBlockStore.ts

```diff
@@ -44,13 +69,44 @@ export const actions: ActionTree<ICustomFlowState, IRootState> = {
       config: {
         prompt: blankPromptResource.uuid,
         choices: {},
+        minimum_choices: undefined,
```

Comment:
Can you give me an update when these new prop should be available from flow-runner ?

Reply:
We've got a JIRA scheduled and can discuss at huddle tomorrow! See: https://votomobile.slack.com/archives/CGGFKUP4Y/p1627585834423100

Reply:
Thanks